lib/cradle.js

```diff
@@ -89,6 +88,8 @@ cradle.Connection = function Connection(/* variable args */) {
 
     if (this.protocol === 'https') this.options.secure = true;
 
+    this.port    = port || (this.options.secure ? 443 : cradle.port);
```

Comment:
Remove the extra spaces after ``this.port``.

Also, we should allow port number 0 here, even though it's "technically" a reserved port. In other words, do something like ``port !== null && port !== undefined ? port : (this.options.secure ? 443 : cradle.port)``.

After that it looks good for merging.